ext/standard/basic_functions.c

```diff
@@ -2586,6 +2586,10 @@ ZEND_END_ARG_INFO()
 ZEND_BEGIN_ARG_INFO_EX(arginfo_is_iterable, 0, 0, 1)
 	ZEND_ARG_INFO(0, var)
 ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_INFO_EX(arginfo_is_countable, 0, 0, 1)
```

Comment:
This should have return type information of `bool` as well.

Reply:
It may also return null on zpp failure, at which point the typehint is more harmful than useful until such a time as we make all zpp failures throwing.

Reply:
Can we please make a push to fix these type-safety related issues in PHP 8? Internal and external function behaviors need to be brought in-line with each other. If we can't even ensure that `callable (mixed): bool` will actually return `bool` if it happens to be an internal function then we have can't reasonably think about type safety at all.

Reply:
So it should be `ZEND_BEGIN_ARG_INFO(arginfo_is_countable, 0)`?

I tried to research what are the differences between `ZEND_BEGIN_ARG_INFO` and `ZEND_BEGIN_ARG_EX` but I didn't found :disappointed:

Reply:
Agreed with @morrisonlevi here, we should eliminate these inconsistency in the near future.